src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+    }
+
+    public Book(Long l, String book1, BigDecimal bigDecimal) {

```

Comment:
The constructor parameters should have more descriptive names. Consider renaming `Long l`, `String book1`, and `BigDecimal bigDecimal` to `Long id`, `String title`, and `BigDecimal price` respectively for clarity.

Reply:
ok, done